dftparse/wien2k/scf2_parser.py

```diff
@@ -0,0 +1,23 @@
+from ..core import BlockParser
+
+
+def _parse_bandgap(line, lines):
+    bandgap = float(line.split()[6])
+    return {
+        'band gap': bandgap,
+        'total energy units': "eV"
+    }
+
+
+base_rules = [
+    (lambda x: ":GAP (global)" in x, _parse_bandgap)
+]
+
+
+class Scf2Parser(BlockParser):
+    """Parser for Wien2k's .scf file"""
```

Comment:
update

Reply:
update what here?

Reply:
This is the parser for the scf2 file, right?